operator/webhook/addons/certmanager/main.py

```diff
@@ -0,0 +1,156 @@
+            for u in annotations.get(
+                "cert-manager.io/subject-organizationalunits"
+            ).split(",")
+        ]
+        if annotations.get("cert-manager.io/subject-organizationalunits", None)
+        is not None
+        else []
+    )
+    countries = (
+        [
+            c.strip()
+            for c in annotations.get("cert-manager.io/subject-countries").split(",")
+        ]
+        if annotations.get("cert-manager.io/subject-countries", None) is not None
+        else []
+    )
+    provinces = (
+        [
+            p.strip()
+            for p in annotations.get("cert-manager.io/subject-provinces").split(",")
+        ]
+        if annotations.get("cert-manager.io/subject-provinces", None) is not None
+        else []
+    )
+    localities = (
+        [
+            l.strip()
+            for l in annotations.get("cert-manager.io/subject-localities").split(",")
+        ]
+        if annotations.get("cert-manager.io/subject-localities", None) is not None
+        else []
+    )
+
+    dns_names = [
+        f"{common_name}.{namespace}.svc.cluster.local",
+        f"{common_name}.{namespace}.svc",
+        f"{common_name}.{namespace}",
+        common_name,
+        f"{name}-actuator.{namespace}.svc.cluster.local",
+    ]
+    dns_names = dns_names + alt_names
+
+    subject = {}
+    if organizational_units:
+        subject["organizationalUnits"] = organizational_units
+
+    if countries:
+        subject["countries"] = countries
+
+    if provinces:
+        subject["provinces"] = provinces
+
+    if localities:
+        subject["localities"] = localities
+
+    cert = {
+        "apiVersion": "cert-manager.io/v1",
+        "kind": "Certificate",
+        "metadata": {
+            "name": f"{name}-certs",
+            "namespace": namespace,
+        },
+        "spec": {
+            "commonName": f"{common_name}.{namespace}",
+            "dnsNames": dns_names,
+            "issuerRef": {
+                "group": "cert-manager.io",
+                "kind": "ClusterIssuer",
+                "name": cluster_issuer,
+            },
+            "keystores": {
+                keystore_type: {
+                    "create": True,
+                    "passwordSecretRef": {
+                        "key": "password",
+                        "name": password_secret_ref_name,
+                    },
+                }
+            },
+            "secretName": f"{name}-certstore",
+            "subject": subject,
+        },
+    }
+
+    return cert
+
+
+def _get_keystore_type(obj) -> str:
+    return obj["spec"]["tls"]["keystore"]["type"]
+
+
+def _get_password_secret_ref_name(obj) -> str:
+    return obj["spec"]["tls"]["keystore"]["passwordSecretRef"]
+
+
+def sync_certificate(body) -> Mapping:
+    # Request API at for DecoratorController at https://metacontroller.github.io/metacontroller/api/decoratorcontroller.html#sync-hook-request
+    _LOGGER.debug("\n\nRequest:\n%s", body)

```

Comment:
Same issue with redundant logging here

Reply:
Fixed

Reply:
Should we remove the other logging statement as well, that logs the reply?

Reply:
Fixed